### ISSUE TITLE
How i can use in lastest natron?

### ISSUE THREAD

**Issue by manuelsongokuh** [tuttleofx/TuttleOFX#467]
I did try use lastest natron not work ..do have other solution ?

**Comment by cchampet** [comment]
Hello,
What do you want to do? Use tuttle plugins on lastest Natron? Natron already provides some tuttle plugins. Which one are missing and you expect? What did not work?

**Comment by manuelsongokuh** [comment]
Last natron has Not include tuttleofx, Mike as off..Not know, i did download tuttleofx with last natron Not works..nothing, you did try last natron ?

**Comment by cchampet** [comment]
I was not aware of that. I tested Natron 2.0.0. Can you share the link of your version?
Before using TuttleOFX plugins in Natron, you need to set an environment variable to tell where are the plugins:
```
export OFX_PLUGIN_PATH=/path/to/your/tuttle/OFX
```
If it is not working, you should open a thread with Naton to discuss about that. Maybe they did some development about this (new way to load plugins...).

**Comment by manuelsongokuh** [comment]
i did add path in preference of natron /home/user/install-tuttle/OFX/
and restart natron is not starting natron... it's crash...so i delete path and natron starts fine..

**Comment by cchampet** [comment]
Do you have a log file?
Something printed in console?

**Comment by manuelsongokuh** [comment]
see result crash:
https://gist.github.com/manuelsongokuh/4e68591b594710f4509e

file natron.config:
https://gist.github.com/manuelsongokuh/109e2c7169d7414cfbe5

**Comment by cchampet** [comment]
What is your platform?

**Comment by manuelsongokuh** [comment]
linux opensuse 13.2 kde4

**Comment by cchampet** [comment]
Ok I will test it on my computer this week.
Did you download [this last tuttle release](https://github.com/tuttleofx/TuttleOFX/releases/tag/v0.12.1)?

**Comment by manuelsongokuh** [comment]
yes that i did download 5 days ago...nothing tuttleofx shows in natron..

**Comment by cchampet** [comment]
I think the problem is: 
Natron depends on dynamic libraries (in its lib/ folder). The Tuttle plugins also depend on dynamic libraries (lib/ folder again). We have some dependencies in common (boot, ffmpeg...), but not with the same version. And this is the problem, and cause the application to crash when the Tuttle plugins are loaded.
I will contact the Natron team to discuss about this issue (actually we already discuss about that). The solution is between less dynamic dependencies for Tuttle plugins, and dependencies with common version.

**Comment by manuelsongokuh** [comment]
ok, thank you for news..i will wait for try with tuttleofx ok?

**Comment by cchampet** [comment]
Ok. Thanks for your feedback again.
The next step for us will be issue #475.

**Comment by manuelsongokuh** [comment]
ok, where i can download snapshot of tuttleofx?

**Comment by cchampet** [comment]
We provide binaries only when we deploy a new version, [like this one](https://github.com/tuttleofx/TuttleOFX/releases/tag/v0.12.1).
There is no snapshot which correspond to the develop branch.

**Comment by manuelsongokuh** [comment]
can you add linux binary OFX for 0.12.2 release?

**Comment by fabiencastan** [comment]
Yes, they are now [available](https://github.com/tuttleofx/TuttleOFX/releases).

**Comment by manuelsongokuh** [comment]
thank you, for now its worked with latest natron!!
but i find many crash with tuttleofx.. i will report all later, for now i'm busy 
thank you

**Comment by olear** [comment]
We started bundling (parts of) TuttleOFX in Natron v1 due to the limited selection of plugins available in Natron at that time, and also since official TuttleOFX builds didn't (and still don't) support CentOS6/RHEL6.

TuttleOFX has now been removed from the Linux builds in Natron v2, and users should use the official builds (if they can). There may be some depends problem, since we use a lot of the same libraries (but probably not the same versions), we confirm to the vfxplatform CY2015.
